src/agent/onefuzz/src/machine_id.rs

```diff
@@ -43,6 +43,14 @@ pub async fn get_ims_id() -> Result<Uuid> {
     Ok(value)
 }
 
+// Here we use the "computer name" terminology from the VMSS documentation: https://learn.microsoft.com/en-us/azure/virtual-machine-scale-sets/virtual-machine-scale-sets-instance-ids#scale-set-vm-computer-name
+// this is the hostname of the compute node.
+pub fn get_computer_name() -> Result<String> {
```

Comment:
just curious. Is computer_name different from machine_name below ?

Reply:
Yes, `machine_name` is `{scale-set-name}_{instance-id}`. But now that you point it out, maybe we should be using this instead of `computer_name` since we will not need to decode it!

Reply:
I’m going to do this instead of the Base-36 stuff.